lib/scrabble.rb

```diff
@@ -1,7 +1,17 @@
+require '../lib/player'
+
 class Scrabble
+  attr_reader :word
 
   def score(word)
-    1
+    @word = word
+    word = Scrabble.new("hello")
+    letters = word
+
+    total = 0
+    letters.each do |letter|
+      total += point_values[letter]
+    end
```

Comment:
This pattern here on lines 11-14 is a good one, it would have worked with two additions. 1) the string being held by the variable word needs to be split up into an array of individual letters. I recommend .chars for that. 2) the letters coming in are lowercase but you need them to be uppercase to match the keys in the hash. I recommend .upcase for that, run before you run .chars  

Put that together and it might look like this: 
letters = word.upcase.chars
letters.each do |letter|
  ...
end